rocky/katalogus/views/plugin_settings_list.py

```diff
@@ -19,24 +19,26 @@ class PluginSettingsListView(SinglePluginView):
 
     def get_plugin_settings(self) -> list[dict[str, Any]]:
         """Gets schema setting with additional info of the value of a setting."""
-        try:
-            if self.plugin_schema is None:
+        if self.organization_member.has_perm("tools.can_view_katalogus_settings"):
+            try:
+                if self.plugin_schema is None:
+                    return []
```

Comment:
I'd swap these around:

```suggestion
        if not self.organization_member.has_perm("tools.can_view_katalogus_settings"):
          return []
        if self.plugin_schema is None:
          return []
        try:
           settings = self.katalogus_client.get_plugin_settings(plugin_id=self.plugin.id)
        except HTTPError:
          ...
          return []
        props = self.plugin_schema.get("properties", [])
        return [for prop in props]  
```

That keeps the logic simpler down the road.